Kickstarter-iOS/Views/Cells/PledgePaymentMethodsCell.swift

```diff
@@ -0,0 +1,90 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class PledgePaymentMethodsCell: UITableViewCell, ValueCell {
+  // MARK: - Properties
+
+  private let layout: UICollectionViewLayout = {
+    UICollectionViewFlowLayout()
+      |> \.scrollDirection .~ .horizontal
+  }()
+
+  private lazy var collectionView: UICollectionView = {
+    UICollectionView(frame: .zero, collectionViewLayout: self.layout)
+      |> \.dataSource .~ self
+  }()
+
+  private lazy var headerStackView: UIStackView = { UIStackView(frame: .zero) }()
```

Comment:
Can we call this `rootStackView`, we tend to do that for the top-most stack view, makes it easier to find 👍